<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_information</class>
 <widget class="QDialog" name="Dialog_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>24</y>
     <width>571</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>订单号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ticket">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>用户id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_userld">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_user">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>航班号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_flight">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>起飞地</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tc">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>到达地</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ac">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>起飞时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_tt">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>到达时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_at">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>状态</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_status">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancle">
        <property name="text">
         <string>取消订单</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>label_ticket</zorder>
  <zorder>label_3</zorder>
  <zorder>label_userld</zorder>
  <zorder>label_5</zorder>
  <zorder>label_user</zorder>
  <zorder>label_7</zorder>
  <zorder>label_flight</zorder>
  <zorder>label_9</zorder>
  <zorder>label_tc</zorder>
  <zorder>label_11</zorder>
  <zorder>label_ac</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>horizontalSpacer_3</zorder>
  <zorder>horizontalSpacer_4</zorder>
  <zorder>label_13</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder>label_tt</zorder>
  <zorder>label_15</zorder>
  <zorder>label_at</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>label_userld</zorder>
  <zorder>horizontalSpacer_6</zorder>
  <zorder>label_2</zorder>
  <zorder>label_status</zorder>
  <zorder>horizontalSpacer_7</zorder>
  <zorder>btn_cancle</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
